src/modules/Device/Device.ts

```diff
@@ -104,22 +96,25 @@ class Device extends TagoIOModule<DeviceConstructorParams> {
   }
 
   /**
-   * Edit  data from device
-   * @param data An array or one object with data to be edited at TagoIO using device token
+   * Edit data in a Mutable-type device.
```

Comment:
My merge with master had conflicts with the `editData` method, but it was only documentation.